From db5f8d03f3f7b75949bf3662970dfab4ca3fcacb Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Sat, 30 Jan 2010 17:53:52 +0100
Subject: [PATCH 2/5] build: fix check-other-sector_sizes

"make check-other-sector_sizes" was running tests of 512-byte sectors
four times, rather than simulating larger sector-sizes for the latter
three iterations.
* Makefile.am (ss-1024 ss-2048 ss-4096): Fix the typo (missing "subst").
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 0e80967..d5a32ef 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -78,7 +78,7 @@ MAINTAINERCLEANFILES += \
 
 .PHONY: ss-1024 ss-2048 ss-4096
 ss-1024 ss-2048 ss-4096:
-	PARTED_SECTOR_SIZE=$(ss-,,$@) $(MAKE) check-recursive
+	PARTED_SECTOR_SIZE=$(subst ss-,,$@) $(MAKE) check-recursive
 
 # Run the regression test suite with different settings,
 # to ensure it works with simulated partition sizes > 512.
-- 
2.1.0

